pkg/vex/vex.go

```diff
@@ -177,7 +177,6 @@ func OpenCSAF(path string, products []string) (*VEX, error) {
 						Status:          StatusFromCSAF(status),
 						Justification:   "", // Justifications are not machine readable in csaf, it seems
 						ActionStatement: just,
-						References:      []VulnerabilityReference{},
```

Comment:
I removed this custom field which is not part of the Minimum elements doc. We can read it later if we need to